web/templates/page/index.html.eex

```diff
@@ -53,7 +53,9 @@
         </td>
         <td class="tc">
           <%= for assignee <- generate_avatar(issue) do %>
-            <img src="https://www.github.com/<%= assignee %>.png" alt="<%= assignee %>'s github avatar'" class="w1plus br2">
+            <a href="https://www.github.com/<%= assignee %>" title="<%= assignee %>">
+              <img src="https://www.github.com/<%= assignee %>.png" alt="<%= assignee %>'s github avatar'" class="w1plus br2">
```

Comment:
This avatar png link is much nicer than what I've been using 👍 

We should be referencing images like this in the future.

(From [here](https://github.com/dwyl/tudo/blob/master/test/controllers/hook_controller_test.exs#L10), https://github.com/shouston3.png would have been so much nicer than https://avatars3.githubusercontent.com/u/15983736?v=4&u=8b6cfb76f2bf2c65dc0b215e179abe1d4cf9c42a&s=400)

Reply:
Yeah I was so surprised when I discovered it! I didn't even know redirect urls _could_ be used as img src.